ci/runtests.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+function setup_root() {
+  mkdir /root/.pip && ln -s /workspace/pip.conf /root/.pip/pip.conf
+  ln -s /workspace/netrc                 /root/.netrc
+  ln -s /workspace/cloudbuild_pypirc     /root/.pypirc
+}
+
+function setup_app() {
+  mkdir /app && cd /app
+  # copy, because we wanna be able to toss out __pychache__ and *.pyc files
+  cp -ar /workspace/gcloud_requests      /app/gcloud_requests
+  cp -ar /workspace/tests                /app/tests
+  # copy, don't link, because we changeit with `sed`
```

Comment:
This can be done using ln -s, because the  it doen't get manipulated. It was a workaround to get another project running. Also can delete the comment